common/common.scss

```diff
@@ -1,5 +1,39 @@
-.__markerjs2_ {
-  z-index: 10000 !important;
+.__markerjs2_, .__cropro__0_ {
+  z-index: 100000 !important;
   position: fixed !important;
 }
 
+.image-annotator-hovering-wrapper {
+  position: absolute;
+  left: 10px;
+  top: 10px;
+  z-index: 1001;
+}
+
+.image-annotator-hovering-wrapper .image-annotator-hovering-button {
+  padding: 5px 10px;
+  border: 1px solid var(--primary);
+  border: none;
+  border-radius: 5px;
+  margin-left: 3px;
+  background: #0087ca;
```

Comment:
Let's use CSS variables instead of hex values. That is blue, so it should be `var(--tertiary)` or `var(--tertiary-{high/medium/low}`